### PR TITLE
Setup Travis CI

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

**Comment by sebapawlak** [comment]
Do we need gradle plugin 0.5.0 here as well?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - test
+  - name: release
+    if: env(RELEASE) == true AND fork == false
+
+jobs:
+  include:
+    - stage: test
+      script: ./gradlew build
+    - stage: release
+      script: ./scripts/travis-release.sh
```
Comment (dagguh) [inline .travis.yml:13]: Gotta add the `./scripts/travis-release.sh` file

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
```
Comment (dagguh) [inline .travis.yml]: Let's add the trailing blank line, just to get rid of the red here ;)

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+stages:
+  - test
+  - name: release
+    if: env(RELEASE) == true AND fork == false
+
+jobs:
+  include:
+    - stage: test
+      script: ./gradlew build
+    - stage: release
```
Comment (dagguh) [inline .travis.yml:12]: Since we're enabling releases here, we should use `gradle-release:0.5.0` to get the proper POM SCM

**Review by dagguh — CHANGES_REQUESTED** [review]

**Review by sebapawlak — DISMISSED** [review]

**Pull request merged**